public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -731,60 +731,60 @@ block style-property-name-dart-diff
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-1')(format=".")
 :marked
-  Some people prefer the `on-` prefix alternative, known as the *canonical form*:
+  Some people prefer the `on-` prefix alternative, known as the **canonical form**:
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-2')(format=".")
 :marked
-  Element events may be the more common targets, but Angular looks first to see if the name matches an event property
+  Element events may be the more common targets, but Angular looks first to see if the name matches the event property
   of a known directive, as it does in the following example:
 +makeExample('template-syntax/ts/app/app.component.html', 'event-binding-3')(format=".")
 
 .l-sub-section
   :marked
-    The `myClick` directive is further described below in the section
-    on [Aliasing input/output properties](#aliasing-io).
+    The `myClick` directive is further described in the section
+    on [aliasing input/output properties](#aliasing-io).
 
 :marked
-  If the name fails to match an element event or an output property of a known directive,
+  If the name fails to match the element event or output property of a known directive,
   Angular reports an “unknown directive” error.
 
   ### *$event* and event handling statements
-  In an event binding, Angular sets up an event handler for the target event.
+  In event binding, Angular sets up an event handler for the target event.
 
   When the event is raised, the handler executes the template statement.
-  The template statement typically involves a receiver that wants to do something
-  in response to the event, such as take a value from the HTML control and store it
-  in a model.
+  The template statement typically involves a receiver, which performs an action
+  in response to the event, such as storing a value from the HTML control
+  into a model.
 
   The binding conveys information about the event, including data values, through
-  an **event object named `$event`**.
+  an **event object named** `$event`.
 
-  The shape of the event object is determined by the target event itself.
-  If the target event is a native DOM element event, the `$event` is a
+  The shape of the event object is determined by the target event.
+  If the target event is a native DOM element event, then `$event` is a
   [DOM event object]( https://developer.mozilla.org/en-US/docs/Web/Events),
   with properties such as `target` and `target.value`.
 
   Consider this example:
 +makeExample('template-syntax/ts/app/app.component.html', 'without-NgModel')(format=".")
 :marked
-  We’re binding the input box `value` to a `firstName` property, and we’re listening for changes by binding to the input box’s `input` event.
+  Here, the input box `value` is bound to the `firstName` property. Changes are being listened for by binding to the input box’s `input` event.
```

Comment:
That's a good start, but the second sentence sounds like a command instead of a description. Maybe:

> This code sets the input box `value` property by binding to the `firstName` property. To listen for changes to the value, the code binds to the input box's `input` event.

(I wasn't sure about _this/the code_ as the actor, but I can't think of anything better. Another possibility was _template_, but that sounded too big/vague.)